<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>251</width>
     <height>384</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="idSuratMasukLabel">
      <property name="text">
       <string>Id Surat Masuk</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="idSuratMasukLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="idAdminLabel">
      <property name="text">
       <string>Id Admin</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="idAdminLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="tanggalMasukLabel">
      <property name="text">
       <string>Tanggal Masuk</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateEdit" name="tanggalMasukDateEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="noSuratLabel">
      <property name="text">
       <string>No Surat</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="noSuratLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="uraianLabel">
      <property name="text">
       <string>Uraian</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="uraianLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="lanjutKeLabel">
      <property name="text">
       <string>Lanjut Ke</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lanjutKeLineEdit"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="dinasInstansiLabel">
      <property name="text">
       <string>Dinas Instansi</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="dinasInstansiLineEdit"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="tandaTerimaLabel">
      <property name="text">
       <string>Tanda Terima</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="tandaTerimaLineEdit"/>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="BtnSimpan">
      <property name="text">
       <string>SIMPAN</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="BtnUbah">
      <property name="text">
       <string>UBAH</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="BtnHapus">
      <property name="text">
       <string>HAPUS</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="tanggalSuratLabel">
      <property name="text">
       <string>Tanggal Surat</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDateEdit" name="tanggalSuratDateEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
